### ISSUE TITLE
Backup to remote does not work (OneDrive)

### ISSUE THREAD

**Issue by Hondelyk** [Kyrodan/KeeAnywhere#429]
I'm using OneDrive to store my keepass database. In KeeAnywhere settings I have both "backup to local" and "backup to Remote" selected. Locally backup is working fine, however in the OneDrive only "main" file is being updated, backup files are not created there. This was working fine until May-June, so I would suspect the latest version to break this feature. If you need some more info please let me know

**Screenshots**
![image](https://github.com/user-attachments/assets/c5ccb112-5416-4bf5-bf57-15b5ff0b1d70)
![image](https://github.com/user-attachments/assets/99fa4a32-6eb3-4880-90a3-b781c9af0320)

**Environment:**
 - OS and Version:  Windows 11
 - KeePass-Version: 2.57
 - KeeAnywhere-Version: 2.1.0

**Comment by lostgit** [comment]
See the solution I reported on [https://github.com/Kyrodan/KeeAnywhere/issues/282](url). 

TL;DR remove the existing OneDrive credential from Tools -> KeeAnywhere and recreate it.

**Comment by ross-at-ormo** [comment]
I think this issue still exists in 2.1.0. Have tried the following to resolve it:
- Removed the old backup files that were still on my OneDrive.
- Removing and re-adding my OneDrive credential multiple times.
- Toggling the remote backup option.
- Changing the number of backups.
